tests/main/interfaces-wayland/task.yaml

```diff
@@ -0,0 +1,53 @@
+summary: Ensure that the wayland interface works
+
+# Only test on classic Ubuntu amd64 systems that have wayland
+systems: [ ubuntu-1*-*64 ]
+
+prepare: |
+    . $TESTSLIB/pkgdb.sh
+    snap install --edge test-wayland
```

Comment:
Our usual pattern for spread helpers is: `test-snapd-*`, i.e. `test-snapd-wayland`.

Reply:
Done